iocore/net/UnixNetVConnection.cc

```diff
@@ -1139,8 +1139,8 @@ UnixNetVConnection::mainEvent(int event, Event *e)
       (write.vio.mutex && wlock.get_mutex() != write.vio.mutex.get())) {
 #ifdef INACTIVITY_TIMEOUT
     if (e == active_timeout)
-#endif
       e->schedule_in(HRTIME_MSECONDS(net_retry_delay));
+#endif
     return EVENT_CONT;
```

Comment:
The e->schedule_in() makes InactivityCop stop running if the e is InactivityCop's callback Event.
because schedule_in() set period to 0.